clients/rospy/src/rospy/topics.py

```diff
@@ -426,7 +426,16 @@ def add_connection(self, c):
             self.connections = new_connections
 
             # connections make a callback when closed
-            c.set_cleanup_callback(self.remove_connection)
+            # XXX: don't clobber an existing callback
```

Comment:
Can you please remove the `XXX: ` form this comment line since it usually indicates a todo.